andela/ah-frontend-space
# #166880067 Setup redux for web application

#### What does this PR do?

Sets up redux for the web application.

#### Description of Task to be completed?

- add a folder for actions
- add a folder for reducers
- add a store.js file

#### How should this be manually tested?
- clone this repo and run ``` npm install``` for dependencies.
- run ```npm start``` in your terminal 
- go to ```http://localhost:8080``` in your browser and inspect using react devtools 
- you will be able to see the store of the application

### Screenshots.
![image](https://user-images.githubusercontent.com/44563961/60594249-35dba180-9dad-11e9-83d2-215dfc5a3a31.png)

#### What are the relevant pivotal tracker stories?

[#166880067](https://www.pivotaltracker.com/story/show/166880067)
